### PR TITLE
update 1

### PR DESCRIPTION
**Pull request by AST-tnbt**

### CODE REVIEW

File: test.cpp [hunk 1]
```diff
@@ -3,5 +3,6 @@ using namespace std;
 int main()
 {
     cout << "abc";
+    cout << "tindeptraibodoi";
```
Comment (nguyenchivi001) [inline test.cpp:6]: code cai loz gi vay

**Review by nguyenchivi001 — COMMENTED** [review]
hoi coin card

File: test.cpp (hunk 1 above)
Comment (vinh0605) [reply test.cpp:6]: coin card

Comment (vinh0605) [inline test.cpp:6]: why

Comment (AST-tnbt) [reply test.cpp:6]: ahihi

**Pull request merged**